File: app/src/main/java/com/song/deviceinfo/ui/hook/HookViewModel.java
package com.song.deviceinfo.ui.hook;

import android.content.Context;

import com.song.deviceinfo.info.HookInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/7/16.
 */
public class HookViewModel extends NormalViewModel {

    public List<Pair<String, String>> getHookInfo(Context context) {
        return HookInfo.getHookInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/hook/HookViewModel.kt
package com.song.deviceinfo.ui.hook

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.HookInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/7/16.
 */
class HookViewModel : NormalViewModel() {
    fun getHookInfo(context: Context?): List<Pair<String, String>> {
        return HookInfo.getHookInfo(context!!)
    }
}
